src/vs/workbench/contrib/interactive/browser/replInputHintContentWidget.ts

```diff
@@ -123,23 +123,32 @@ export class ReplInputHintContentWidget extends Disposable implements IContentWi
 		const shiftEnterConfig = this.configurationService.getValue(InteractiveWindowSetting.executeWithShiftEnter);
 
 		if (shiftEnterConfig) {
-			const keybinding = keybindings.find(kb => kb.getLabel() === 'Shift+Enter');
+			const keybinding = keybindings.find(kb => {
+				const chords = kb.getDispatchChords();
+				return chords.length === 1 && chords[0] === 'shift+Enter';
+			});
 			if (keybinding) {
 				return keybinding;
 			}
 		} else {
-			let keybinding = keybindings.find(kb => kb.getLabel() === 'Enter');
+			let keybinding = keybindings.find(kb => {
+				const chords = kb.getDispatchChords();
+				return chords.length === 1 && chords[0] === 'Enter';
+			});
 			if (keybinding) {
 				return keybinding;
 			}
 			keybinding = this.keybindingService.lookupKeybindings('python.execInREPLEnter')
```

Comment:
Why are we looking up `python.` commands in core?

Reply:
python shadows the command for the REPL, this hint doesn't work correctly if we don't look it up, and the point is to make this clear for python/jupyter users